README.md

```diff
@@ -1,7 +1,8 @@
-skopeo [![Build Status](https://travis-ci.org/containers/skopeo.svg?branch=master)](https://travis-ci.org/containers/skopeo)
+<!--- skopeo [![Build Status](https://travis-ci.org/containers/skopeo.svg?branch=main)](https://travis-ci.org/containers/skopeo)
```

Comment:
@cevich or @edsantiago do you know who set this up?  There are a number of these labels scattered throughout the projects, but most are pointing to "master" and I think are giving false positives.  If I change this to main, it says the CI failed all the time.  I suspect there's a bit to flip somewhere in travis to change from "master" to "main", but I'm not sure where that is.  Thoughts?

Reply:
This is cargo-culted all the way back from https://github.com/runcom/skopeo/pull/1 .

We have also migrated away from Travis about two years ago.

So I think it’s clear that we don’t use this badge for anything ourselves, and I can’t see what end users would do with that information.

IMHO it’s mostly advertising for that service that brings our project no value, and I propose just dropping that entirely.

Reply:
Dropping it is fine.  Or swap it to the badge provided by Cirrus: [![Build Status](https://api.cirrus-ci.com/github/containers/skopeo.svg)](https://cirrus-ci.com/github/containers/skopeo)

`[![Build Status](https://api.cirrus-ci.com/github/containers/skopeo.svg)](https://cirrus-ci.com/github/containers/skopeo)`

Reply:
Are you looking at that badge, or do you know of anyone that is?

Reply:
I don't look at frequently enough to justify it's existence.  I doubt anybody really cares if it's there or what color it is.  It's only useful if both:

1. Somebody with CI knowledge notices it turn red.
2. Fixes (or tells somebody about) it.

For this repo, likely 99.999% of the time if it turns red, it's a flake.  The key being, many/most/all PRs merge in-sequence here.  That's not the case for Buildah/Podamn, where I do look at the badge/color maybe once a week.

Reply:
The way I’m thinking about it:
- for PRs, the PR submitter is going to notice in the PR (though does fail, pretty badly, from time to time — the reviewer thinks the submitter will automatically take care of the problem, the reviewer is unaware)
- for non-PR CI status, we have the Podman-monitor mailing list, where failures are pushed to us, they don’t require actively looking for a failure.

Reply:
> for non-PR CI status, we have the Podman-monitor mailing list

Oh good point, yes we pay much more close attention to that.

Reply:
I didn't even think about the Travis removal, ugh.  I've removed the tag